src/main/java/com/twilio/oai/api/ApiResourceBuilder.java

```diff
@@ -134,51 +140,48 @@ protected Map<String, Object> mapOperation(CodegenOperation operation) {
     }
 
     private void applyListOperation(CodegenOperation operation, Map<String, Object> operationMap, HttpMethod httpMethod) {
-
         if(!operation.vendorExtensions.containsKey("x-ignore")) {
             if (httpMethod == HttpMethod.POST) {
                 operationMap.put("x-name", API_OPERATION_CREATE);
-                operationMap.put("x-is-create-operation", true);
             } else if (httpMethod == HttpMethod.GET) {
                 operationMap.put("x-name", API_OPERATION_READ);
                 metaAPIProperties.put("hasReadOperation","true");
-                operationMap.put("x-is-read-operation", true);
             }
+            operationMap.put("x-resource-name", getApiName() + "ListInstance");
```

Comment:
We can also have "fetch" operation here. For that case, the "x-name" will have the value = "Read"

Reply:
These were added for Node. Fine with taking them out, but they're not used by other languages.